assets/javascripts/modules/autoCompleteFactory.js

```diff
@@ -2,18 +2,28 @@ require('jquery');
 var autoComplete = require('./autoComplete.js');
 
 var createAutoCompleteCountries = function () {
```

Comment:
want to also add an issue around removing this deprecated work? and playing any html updates in service frontends?